tests/contexts/index.js

```diff
@@ -0,0 +1,35 @@
+const invalidContexts = {
+  veresOne: {
+    url: 'https://w3id.org/veres-one/v1',
+    value: require('./veresOnev1.jsonld')
+  },
+  did: {
+    url: 'https://w3id.org/did/v0.11',
+    value: require('./didv0.jsonld')
+  },
+  valid: {
+    url: 'https://w3id.org/credentials/v1',
+    value: require('../../lib/contexts/credentials-v1.js')
+  },
+  invalidId: {
+    url: 'https://invalid-id.org',
+    value: require('./invalid_id.jsonld')
+  },
+  nullVersion: {
+    url: 'https://null-version.org',
+    value: require('./null_version.jsonld')
+  },
+  nullId: {
+    url: 'https://null-id.org',
+    value: require('./null_@id.jsonld')
```

Comment:
I'd be better to avoid any filename character set issues with `@` and just use a simple name like `null_id.json`